webdriver/tests/interface.html

```diff
@@ -1,49 +1,23 @@
 <!doctype html>
 <meta charset=utf-8>
-
+<link rel="help" href="https://w3c.github.io/webdriver/webdriver-spec.html">
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script src=/resources/WebIDLParser.js></script>
 <script src=/resources/idlharness.js></script>
 
-<script type=text/plain class=untested>
-[Exposed=Window]
-interface Navigator {
-  // objects implementing this interface also implement the interfaces given below
-};
-</script>
-
-<script type=text/plain>
-Navigator includes NavigatorAutomationInformation;
-
-interface mixin NavigatorAutomationInformation {
-  readonly attribute boolean webdriver;
-};
-</script>
-
 <script>
-"use strict";
-
-test(() => assert_idl_attribute(navigator, "webdriver"), "navigator.webdriver is present");
-
-// When test is run in automation navigator.webdriver is likely to
-// be true because WebDriver controls the browser instance.  To that
-// extent, this test is a bit special.  It should also be possible to
-// run the test manually, when WebDriver is not active, and so either
-// true/false outcome is OK.
-if (navigator.webdriver) {
-  test(() => assert_true(navigator.webdriver), "navigator.webdriver is true when webdriver-active is set");
-} else {
-  test(() => assert_false(navigator.webdriver), "navigator.webdriver is false when webdriver-active is not set");
-}
-
-var idls = new IdlArray();
-for (let node of [...document.scripts].filter(({type}) => type == "text/plain")) {
-  if (node.className == "untested") {
-    idls.add_untested_idls(node.textContent);
-  } else {
-    idls.add_idls(node.textContent);
-  }
-};
-idls.test();
+  "use strict";
+
+  promise_test(async () => {
```

Comment:
I get this when running the test locally:

> JavaScript error: file:///home/ato/src/wpt/webdriver/tests/interface.html, line 12: ReferenceError: promise_test is not defined

Reply:
Looks like you're trying to run the test from file://. Please run `wpt run` and use http://web-platform.test:8000/webdriver/tests/interface.html instead.

Reply:
That’s not what I want to test. I explicitly want the test to pass when loading it manually with WebDriver disabled.

Reply:
Oops, I meant `wpt serve`. testharness.js tests can't be loaded over file:// is the important point.

Reply:
As foolip said, from the `wpt` root, run `./wpt serve` then head to http://localhost:8000/webdriver/tests/interface.html

Reply:
Thanks, I can confirm it passes as expected both with and without Marionette enabled in Firefox.